cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml

```diff
@@ -228,21 +239,25 @@ Parameters:
           - "Running"
           # - "Hibernated"
       AsgWarmPoolMaxGroupPreparedCapacity:
-        Type: String
-        Description: >-
-          Specifies the total maximum number of instances that are allowed to be in the warm pool or in any state except Terminated for the Auto Scaling group. 
-          Ignored when 'Auto-scaling Group Warm Pool Enabled' is No
-        AllowedPattern: '^(.*|[0-9]+)$'
+        Type: Number
```

Comment:
I remember not using Type:Number because the validation algos within AWS CFT will fail if it is a Number and left empty, which might be the case if the user chooses not to use warm pool, for all warm pool based params

Reply:
let me know if this is required to rollback